modules/t/test-genome-DBs/multi/metadata/organim.txt

```diff
@@ -0,0 +1 @@
+170	9606	0	9606	homo_sapiens	Homo_sapiens	Human	Homo sapiens	NULL	NULL	NULL	NULL
```

Comment:
Typo in filename, 'organism.txt' not 'organim.txt'.

Reply:
Well spotted, thanks